Добавила классы и звездный рейтинг в index.html

Punkcakeee opened the pull request.

Punkcakeee committed:
dlk_разметка

Добавила разметку dlk main

Punkcakeee committed:
Merge branch 'dl-academy-edu:main' into main

Punkcakeee committed:
Update index.html

Punkcakeee committed:
Внесла правки в index.html

Внесла правки в index.html (второе дз)

Punkcakeee committed:
внесла правки в index.html

Punkcakeee committed:
добавила классы к index.html

добавила классы к index.html + переделала звездный рейтинг в подвале

Yan1993-Dev commented on index.html:
```diff
@@ -0,0 +1,147 @@
+        </section>
+        <section class="trust-info">
+            <img src="/" alt="woman-in-dress">
+            <div class="info-ctg">
```
ctg - что это за слово? я не понял. Именовать классы, элементы нужно более явно

Punkcakeee committed:
update all pages

Сделала разметку и проставила классы на всех страницах. Далее приступлю к картинкам и css
